_health-care/_js/components/HealthCareApp.jsx

```diff
@@ -275,12 +283,53 @@ class HealthCareApp extends React.Component {
           external: this.getExternalData(this.state.applicationData, statePath)
         });
       });
+    }
 
-      if (this.props.location.pathname === '/review-and-submit') {
-        buttons = (<SubmitButton onButtonClick={this.handleSubmit}/>);
-      } else {
-        buttons = (<ContinueButton onButtonClick={this.handleContinue}/>);
-      }
+    // Check which section the user is on and render the correct ProgressButtons.
+    const lastSectionText = (this.getUrl('back')) ? this.getUrl('back').split('/').slice(-1)[0].replace('-', ' ') : '';
```

Comment:
I think the `replace` is only replacing the first instance of `-` and not all of them. I think instead you want `.replace(/-/g, ' ')`.